### PR TITLE
docs: renew the global resource control rfc

### PR DESCRIPTION
**Pull request by glorv**
### What problem does this PR solve?

Issue Number: ref #38825
Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: docs/design/2022-11-25-global-resource-control.md [hunk 1]
```diff
@@ -19,151 +21,184 @@ Moreover, this resource control mechanism can be used by Resource Manage in the
...
+### Request unit Model
 
-**WRU:** the request unit. Used to measure the write request resource consumption.
+Here is to define a cost model relative to the actual cluster resource capacity. Each request will consume the resource capacity. The TiDB layer will cost CPU and memory for different SQL requests, and then SQL requests will transform into the KV requests. These KV requests include different types, requiring the resource of CPU/IO/Memory. All in one, our abstraction unit needs to be mapped to the resources of the cluster, including CPU/IO/Memory, and even the network. For now, we only consider two factors: CPU and IO.
 
-$$RRU=\alpha_w + \beta_w * ReadBytes + \gamma_w * CpuSecs$$
+$$RU=\alpha_r + \beta_r * IOBytes + \gamma_r * CpuSecs$$
 
-The values of the coefficients α,β,γare different between `RRU` and `WRU`, and here require datasets from various tests then regress and fit a solution(values) that best maps the actual resource usage.   
+The values of the coefficients α,β,γ can be different in different environment, they require datasets from various tests then regress and fit a solution(values) that best maps the actual resource usage.
```
Comment (Connor1996) [inline docs/design/2022-11-25-global-resource-control.md:76]:
```suggestion
The values of the coefficients α,β,γ vary in different environment, they require datasets from various tests then regress and fit a solution(values) that best maps the actual resource usage.
```

**Comment by nolouch** [comment]
maybe we can replace the overview picture with:
image

**Review by nolouch — APPROVED** [review]

File: docs/design/2022-11-25-global-resource-control.md [hunk 2]
```diff
@@ -17,153 +19,191 @@ Moreover, this resource control mechanism can be used by Resource Manage in the
...
 
 ### Global Quota Control(- Global admission control)
+
 The global admission control can be an independent module that is easy to use and compiled by TiDB. And also, it's possible to be an independent service. Because its local controller part may be installed to the SQL layer first, or it may be moved from the SQL layer to the proxy layer with the evolution. And it also will be related to the billing system in the future. Therefore, it should be as decoupled as possible.
-The subsystem can be enabled and disabled. Once enabled, it may bring some performance regression (1% ~5% from TopSQL) due to the statistics and control overhead.
+The subsystem can be enabled and disabled. Once enabled, it may bring some performance regression (1%~5% from TopSQL) due to the statistics and control overhead.
+
 Overview
+
 ![global quota control.png](imgs/global-quota-control.png)
```
Comment (nolouch) [inline docs/design/2022-11-25-global-resource-control.md:64]: Maybe we can remove this overview

**Review by Connor1996 — APPROVED** [review]
LGTM

**Review by nolouch — APPROVED** [review]

**Review by nolouch — COMMENTED** [review]
lgtm

**Comment by glorv** [comment]
@BornChanger PTAL

**Pull request merged**